src/modules/announcements/pages/services.jsx

```diff
@@ -0,0 +1,3 @@
+export const Services = () => {
```

Comment:
I think it would be easier to maintain them if any single view would be a different module. Especially that they don't have much in common with `announcements`.

Reply:
I think it should stay in announcements module because on this page we will show anncouments from this category. I think we will use reusable components from this module